lib/diaspora/federation/entities.rb

```diff
@@ -0,0 +1,258 @@
+module Diaspora
+  module Federation
+    module Entities
+      def self.build(entity)
+        case entity
+        when AccountDeletion then account_deletion(entity)
+        when Comment then comment(entity)
+        when Contact then contact(entity)
+        when Conversation then conversation(entity)
+        when Like then like(entity)
+        when Message then message(entity)
+        when Participation then participation(entity)
+        when Photo then photo(entity)
+        when PollParticipation then poll_participation(entity)
+        when Profile then profile(entity)
+        when Reshare then reshare(entity)
+        when Retraction then build_retraction(entity)
+        when StatusMessage then status_message(entity)
```

Comment:
Mh, so this is essentially the same list and mapping we have in the glue code earlier?